Create temp code for ship movement and firing

albinopapa opened the pull request:
Added the files to the solution
Added some files to .gitignore so it may clash with your .gitignore when/if you merge with yours
Added some test code for movement and firing

There aren't any utility structs like Vector just a bunch of loose variables

Your mission is to review the code, run it and see if you like it.
Create the Vector class and replace all the X/Y vars with Vector objects.
overload the +, - and * operators in the Vector struct.
(Note: The +/- operators take a single const Vector & for only parameter, while the * operator should only take in a float for it's only parameter, both return a Vector.)

albinopapa committed:
Added abel's files to the solution and added some files and folders to the .gitignore file.

albinopapa committed:
Delete the : from the empty constructors
Add .suo extension to .gitignore

albinopapa committed:
Copy over temp code from other project

albinopapa committed:
Test

albinopapa committed:
test

adabo reviewed (approved):
Looks good to me. Not that I would have much to say anyway, as I'm just the padawan, Albinwan Kenobi ;)

Commit:
Merge branch 'dev_abel' into dev_josh

albinopapa committed:
more files added to .gitignore

albinopapa committed:
Merge branch 'dev_josh' of https://github.com/adabo/Malaga.git

albinopapa committed:
Fix start stop time by putting start before all frame updates
Change fire rate from .0025f to .1f

albinopapa committed:
Add malagaMKII.txt to solution just to make it easier to open and read on demand

albinopapa committed:
Add Shield class
Add EnemyHoming class
Start adding body to existing classes

albinopapa committed:
Add operator== and operator!= to Colors.h for comparisons
Create Wic class to load images
Add Wic object to Game, must pass to Bitmap to create images
Create Image base class for loading images
Create Bitmap child of Image to draw images
Modify Text class to use C++ functions
Modify Text class to make it more lean, use Color class instead of enums and individual components (r,g,b)

albinopapa committed:
Add Font h/cpp files to project

albinopapa committed:
Remove references to deleted functions and members
Change member accessor from . to -> for Text::font
Change Text::color to Text::active_color

albinopapa committed:
Add example commented code for operator overloading
Copy player input code from Game.cpp to Player.cpp, waiting for Abel to make ship class.
Change input method from Mouse to Keyboard in Player.h
Add SetXY function to Text class
